spec/classes/init_spec.rb

```diff
@@ -497,6 +497,17 @@
             end
           end
 
+          context 'cvmfs_cache_base set to non-defaults' do
+            let(:params) do
```

Comment:
You can use the `super()` trick used in the same file to extend the minimal parameters.